<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>채팅방</string>
  </property>
  <property name="styleSheet">
   <string>
    QMainWindow {
        background-color: #f2f4f9;  /* 파스텔톤 배경색 (연한 파스텔 블루) */
    }
    QPushButton {
        background-color: #7cb5e8;  /* 파스텔 블루 버튼 */
        border: 1px solid #4c8bf5;
        border-radius: 8px;
        color: white;
        font-weight: bold;
        padding: 10px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #4c8bf5;  /* 마우스를 올렸을 때 파스텔 블루에서 조금 더 진한 파란색으로 변경 */
    }
    QLineEdit {
        border: 1px solid #b5c9e2;  /* 밝은 파스텔 블루 테두리 */
        border-radius: 5px;
        padding: 10px;
        font-size: 14px;
        background-color: #ffffff;  /* 흰색 배경 */
    }
    QListWidget {
        background-color: #ffffff;  /* 채팅 메시지 리스트 배경색 흰색 */
        border: 1px solid #b5c9e2;  /* 파스텔 블루 테두리 */
        border-radius: 5px;
        padding: 10px;
        font-size: 14px;
        color: #333;  /* 텍스트 색상은 어두운 회색 */
    }
    QListWidget::item {
        padding: 8px;
        border-bottom: 1px solid #f2f4f9;  /* 항목 간 구분선 색상 */
    }
    QListWidget::item:selected {
        background-color: #7cb5e8;  /* 항목 선택 시 파스텔 블루 */
        color: white;
    }
    QLabel {
        font-size: 16px;
        font-weight: bold;
        color: #333;
    }
    QMenuBar {
        background-color: #f2f4f9;  /* 메뉴 바 배경색 */
        color: #333;
    }
    QStatusBar {
        background-color: #f2f4f9;  /* 상태 바 배경색 */
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="chatMessagesList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>450</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>메시지를 입력하세요...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QListWidget" name="userListWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>59</y>
      <width>120</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QListWidget {
          background-color: #f9f9f9;  /* 사용자 목록 배경 (밝은 회색) */
          border: 1px solid #b5c9e2;  /* 파스텔 블루 테두리 */
          border-radius: 5px;
          font-size: 14px;
      }
      QListWidget::item {
          padding: 8px;
      }
      QListWidget::item:selected {
          background-color: #7cb5e8;  /* 선택 시 파스텔 블루 */
          color: white;
      }
     </string>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveRoomButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>채팅방 나가기</string>
    </property>
   </widget>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
